HtmlCssSample.css

```diff
@@ -0,0 +1,188 @@
+body {
```

Comment:
Instead of styling the html component directly, use custom class names
eg:- instead of styling input tag directly, provide a class name to that input and apply style.

Reply:
Resolved